Feature/12 aviso modificacion voto

manmunesp12 opened the pull request:
Notificación para informar al usuario que ya ha votado y confirmación de edición de voto.

dancasnar1 committed:
feat: Avance en el redireccionamiento para avisar de que un usuario ya ha votado en esa votación.

dancasnar1 committed:
feat: Añadida la ventana emergente en caso de haber votado en esa votación.

manmunesp12 committed:
feat: Optimización del aviso de edición de voto.

manmunesp12 committed:
fix: Solucionado el problema presentado a la hora de intentar modificar el voto más de dos veces en la misma sesión.

Javitoox reviewed (changes requested):
Los cambios requeridos son los siguientes:
- Dependencias en el archivo "requirements.txt".
- Usar mongodb.
- Buscar alternativa a la librería de Tkinter ya que no es compatible con wsl.

dancasnar1 committed:
fix: Cambia el archivo local_settings.py.

Se ha modificado el archivo local_settings.py para tenerlo actualizado con la rama main.

dancasnar1 committed:
Merge branch 'main' into feature/12-aviso_modificacion_voto

dancasnar1 committed:
feat: Añade la ventana de aviso de edición de voto.

Se ha modificado la ventana emergente debido a incompatibilidades con Tkinter, la funcionalidad nueva sigue siendo que surja una ventana emergente en caso de que ya hayas votado.

Co-Authored-By: Manuel Muñoz España <manmunesp1@alum.us.es>

NoSoyDani commented:
Se muestra el aviso y se registra adecuadamente la edición del voto.

NoSoyDani reviewed (approved):
La funcionalidad se ha implementado correctamente.

Commit:
Merge branch 'main' into feature/12-aviso_modificacion_voto

dancasnar1 committed:
chore: Añade librería en el archivo requirements.txt

Se ha añadido la librería easygui para implementar la funcionalidad de la ventana emergente en caso de haber votado.

Commit:
chore: Elimine incompatibilidades

Se ha actualizado la versión de la libreria easygui a la versión más
actualizada con la idea de eliminar las incompatibilidades encontradas en
Mac y wsl, junto a pequeños cambios.
